code-of-conduct/guide-code-of-conduct-response-plan.md

```diff
@@ -0,0 +1,188 @@
+You may change these terms if you'd rather use different ones.
+If you do change the terms, make sure you find and replace all instances of the old term in the document.
+
+
+### About the "Handling incident reports" section
+
+The *Handling incident reports* section includes several subsections.
+It makes up the main body of this document and explains how you will handle incident reports.
+
+The remediation process explained in this template was based on the Mozilla process for handling incidents.
+You can feel free to adapt the process for your community if needed.
+Whatever you do, we want you to truly make this process your own.
+Take time to really think through the logistics of how to make this process work for your community.
+
+:information_source: All of the sections in the Code of Conduct response plan template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text. If you use the boilerplate text, make sure you replace the project name placeholders with your own project name.
+
+
+### About the "Overview" section
+
+The *Overview* section provides a brief overview of the different phases of handling a Code of Conduct incident.
+
+
+### About the "Listen" section
+
+The *Listen* section explains the policy and procedures for meeting with a Code of Conduct incident reporter to gather information about the Code of Conduct violation.
+
+At the end of this phase, the investigating moderator should fill out a [Code of Conduct incident record](code-of-conduct-incident-record.md).
+
+
+### About the "Triage" section
+
+The *Triage* section provides guidelines for assigning an initial severity and impact assessment.
+
+- **Severity** refers to the overall seriousness of the behavior and the risk that behavior will be repeated.
+- **Impact** refers to how public the incident was and the number of community members who were or who could have been impacted by the incident, especially members of marginalized communities.
+
+You may use your own assessment categories and definitions if preferred.
+
+
+### About the "Recommend" section
+
+The *Recommend* section provides guidelines for recommending a suggested response to the Code of Conduct incident.
+
+
+### About the "Respond" section
+
+The *Respond* section explains how to meet with the accused individual to deliver the suggested response and behavior modification plan.
+
+At the beginning of this phase, the investigating moderator should fill out the [Code of Conduct remediation record](code-of-conduct-remediation-record.md).
+
+
+### About the "Resolve" section
+
+The *Resolve* section explains how to conclude the Code of Conduct investigation by filing the necessary paperwork for record-keeping purposes.
+
+It is important to file this documentation to enable the community moderators to identify and prevent potential repeated patterns of abuse in the community.

```

Comment:
Here's this use of "file" again. I tripped over it a few times in this document set, but I have commented in those other places.

Reply:
Changed to "record and store."